flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/copy/HiveParserRowResolver.java

```diff
@@ -109,6 +133,17 @@ public void put(String tabAlias, String colAlias, ColumnInfo colInfo) {
         }
     }
 
+    private void keepAmbiguousInfo(String colAlias, String tabAlias) {
```

Comment:
Is it guaranteed that the `colAlias` and `tabAlias` are lowercase? Should we lower again in the method?

Reply:
The `colAlias` may not be lowercase. But it's fine since it's used to track the ambiguous column and we will compare the call [colAlias.equalsIgnoreCase(cmapEnt.getKey())](https://github.com/apache/flink/pull/19017/files#diff-f1a47219ad2140e4127bbdb048daceec042945675725f514d4f04fb3d437957aR516)  to check whether there's  ambiguous column. 
So, we can still find the ambiguous column whether the `colAlias` in the map `ambiguousColumns` is lowercase or not.